src/odemis/util/peak.py

```diff
@@ -318,14 +368,110 @@ def estimateFitTime(self, data):
         return len(data) * 10e-3  # s
 
 
-def Curve(wavelength, peak_parameters, offset, type='gaussian'):
+def peak_to_energy(pos, width, amplitude):
+    """
+    Converts the peaks to energy domain.
+    Args:
+        pos: the center position of the peak in 'm'
+        width: the width of the peak in 'm'
+        amplitude: the amplitude of the peak in space domain
+
+    Returns: peak parameters as (pos, width, amplitude) in energy domain
+
+    """
+    amplitude_e = (amplitude * pos ** 2) / constant
+    width_e = constant * ((1 / (pos - width / 2)) - (1 / (pos + width / 2)))
+    pos_e = constant / pos
+    return pos_e, width_e, amplitude_e
+
+
+def peak_to_wavelength(pos_e, width_e, amplitude_e):
+    """
+    Converts the peaks to space domain.
+    Args:
+        pos_e: the center position of the peak in 'eV'
+        width_e: the width of the peak in 'eV'
+        amplitude_e: the amplitude of the peak in energy domain
+
+    Returns: peak parameters as (pos, width, amplitude) in space domain
+
+    """
+    width = constant * ((1 / (pos_e - width_e / 2)) - (1 / (pos_e + width_e / 2)))
+    pos = constant / pos_e
+    amplitude = (amplitude_e * constant) / pos ** 2  # pos should be in space domain for the reverse Jacobian
+    return pos, width, amplitude
+
+
+def calc_width(initial_width, wavelength):
+    """
+    Calculates a rough estimation of the width by multiplying the initial width by the 1/10 of the wavelength range.
+    Args:
+        initial_width: a parameter which will be multiplied by the ratio peak width of wavelength
+        wavelength: the list of wavelengths in "m" or "px
+
+    Returns: peak width estimation
+
+    """
+    peak_width_ratio = (wavelength[-1] - wavelength[0]) / 10
+    return initial_width * peak_width_ratio
+
+
+def applyJacobian_x(wavelength):
```

Comment:
non consequent naming with functions above, first ones are snake_case then this one uses camelCase**_x**. Functions after this have the same